draft-demarco-oauth-status-attestations.md

```diff
@@ -404,7 +405,13 @@ Credential Issuer MUST respond with an HTTP Response with the status
 code set to 200 and the `status_assertion_responses` array with the related
 Status Assertion Error object.
 
-Below a non-normative example of a Status Assertion Error object in JWT format, with the headers and payload represented in JSON and without applying the signature.
+The Status Assertion Error MUST NOT be presented or provided to a Verifier,
+the only audience of the Status Assertion Error is the Holder of the Credential
+that has requested the Status Assertion. Therefore,
+the Status Assertion Error MUST NOT contain the parameter `aud`.
```

Comment:
@peppelinux please help me to understand this logical implication (Therefore,...). I agree that the response MUST NOT to be presented to a Verifier, but saying that it MUST NOT contain the aud parameter seems to me strong. The logical implication is that
- the aud MAY be used
- if it is used MUST be only the client 
- the client MUST check that the aud parameter only contains the client.

In this way we force the response to be not presented to third party and the aud parameter could also be used as security check. WDYT?

Reply:
we were moved from a json error message to a jwt. Therefore I didn't see the value of adding so much requirements for the data schema of the errors.

I tend to reduce the options (may) aiming to give clear paths to the implementers. However, this requirement to keep out aud from the dataset is not critical to me. I would therefore remove the MUST NOT and not include at the same time the `aud` parameter definition in the assertion error data set

Reply:
@fmarino-ipzs do you agree? Do I have to remove that MUST NOT to get this PR merged?

Reply:
I get your point. I think that forcing third parties to not consume the response jwt can be properly done by using aud parameter as it is stated in the [RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3). But in this PR you are explicitly saying that aud claim MUST NOT to be used and this seems a little bit strange to me..
Anyway it is not critical for me as well so I'm going to approve this PR and we can came to this point later if you think it is valuable